plugins/modules/ecs_taskdefinition.py

```diff
@@ -929,7 +929,7 @@ def fetch():
         return list(
             sorted(
                 [
-                    self.ecs.describe_task_definition(taskDefinition=arn)["taskDefinition"]
+                    self.ecs.describe_task_definition(aws_retry=True, taskDefinition=arn)["taskDefinition"]
```

Comment:
No objection to the change, but why only this call?

Reply:
No particularly good reason. I didn't fully understand the previous state of affairs, with the retry mechanism used by some calls but not others (or why it needs to be explicitly enabled) - so I opted for the minimal change with minimal risk. _This_ call is in a loop, so it looks more likely to be throttled (and also it's what the traceback showed).

2 calls to `self.ecs` remain without `aws_retry=True`, in lines 919 and 940. Would you like me to add `aws_retry=True` there as well?

Reply:
If you're willing, Yes please.  aws_retry is well tested I'm not worried about adding it.

Reply:
Sure! Done.